modules/auth/views/Login.vue

```
@@ -93,20 +93,14 @@
 <script setup lang="ts">
 import { ref } from 'vue'
 import { useRules } from '~/composables/useRules'
-import { useAuthWrapper } from '../composables/useAuthWrapper'
-import { useAuthStore } from '../store/auth.store'
+import { useAuth } from '../composables/useAuth'
 
 const router = useRouter()
```

ESLint

[error] 'useRouter' is not defined.

(no-undef)